<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_logo_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Safatique logo (2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="ProductsTableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>731</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <column>
     <property name="text">
      <string>Product Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ProductID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total Price</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_Checkout">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Checkout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteItem">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backToCatalogue">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to Catalogue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-40</y>
      <width>901</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>backgroundDBMS.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Payable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TotalAmount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TotalAmt</string>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>label_logo_7</zorder>
   <zorder>ProductsTableWidget</zorder>
   <zorder>pushButton_Checkout</zorder>
   <zorder>pushButton_deleteItem</zorder>
   <zorder>pushButton_backToCatalogue</zorder>
   <zorder>label_username_5</zorder>
   <zorder>label_TotalAmount</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
